test/coverlet.core.tests/InstrumenterHelper.cs

```diff
@@ -191,7 +191,9 @@ public static CoverageResult GetCoverageResult(string filePath)
             using (var result = new FileStream(filePath, FileMode.Open))
             {
                 CoveragePrepareResult coveragePrepareResultLoaded = CoveragePrepareResult.Deserialize(result);
-                Coverage coverage = new Coverage(coveragePrepareResultLoaded, new Mock<ILogger>().Object, new InstrumentationHelper(new ProcessExitHandler(), new RetryHelper(), new FileSystem()));
+                var fileSystem = new Mock<IFileSystem>();
+                fileSystem.Setup(x => x.Exists(It.IsAny<string>())).Returns(true);
+                Coverage coverage = new Coverage(coveragePrepareResultLoaded, new Mock<ILogger>().Object, new InstrumentationHelper(new ProcessExitHandler(), new RetryHelper(), fileSystem.Object), fileSystem.Object);
```

Comment:
ditto move under `InstrumentationHelper _instrumentationHelper` and reuse same ref.